plugins/aladino/actions/assignCodeAuthorReviewers.go

```diff
@@ -159,3 +189,18 @@ func isUserValidAssignee(assignees []*codehost.User, username string) bool {
 	}
 	return false
 }
+
+func getTotalRequiredRandomReviewers(reviewers []string, requiredNumberOfReviewers int) []string {
+	if requiredNumberOfReviewers > len(reviewers) {
+		requiredNumberOfReviewers = len(reviewers)
+	}
+
+	randomReviewers := make([]string, requiredNumberOfReviewers)
+	for i := 0; i < requiredNumberOfReviewers; i++ {
+		randIndex := rand.Intn(len(reviewers))
+		randomReviewers[i] = reviewers[randIndex]
+		reviewers = append(reviewers[:randIndex], reviewers[randIndex+1:]...)
+	}
+
+	return randomReviewers
+}
```

Comment:
Suggestion for better reading.

```suggestion
func getRandomReviewers(reviewers []string, total int) []string {
	if total >= len(reviewers) {
		return reviewers
	}

	selectedReviewers := make([]string, total)

	for i := 0; i < total; i++ {
		randIndex := rand.Intn(len(reviewers))
		selectedReviewers[i] = reviewers[randIndex]

		reviewers = append(reviewers[:randIndex], reviewers[randIndex+1:]...)
	}

	return selectedReviewers
}
```